musicname/ze_diablo_v5_1.cfg

```diff
@@ -0,0 +1,37 @@
+"music"
+{
+  //"belial_theme.mp3"                          ""
+    "diablo_azmo_real_boss_music.mp3"           "South Central -The Day I Die (The Qemists Remix)"
+    "diablo_boss_music.mp3"                     "Diablo III OST - And The Heavens Shall Tremble"
+    "diablo_bridge_ex2.mp3"                     "Kayzo - CYBER DETHHH (Vastive Remix)"
+    "diablo_butcher_music_ex.mp3"               "Linkin Park - Points Of Authority"
+    "diablo_butcher_song.mp3"                   "George Shaw - I'm Coming Over"
+  //"diablo_ex_ending_music.mp3"                ""
+    "diablo_ex1_music1_loud.mp3"                "The Prodigy - Invisible Sun"
+    "diablo_ex1_music3_diff.mp3"                "Pendulum - The Catalyst"
+    "diablo_ex2_leophase1.mp3"                  "Black Sun Empire - Bitemark"
+    "diablo_ex2_music1.mp3"                     "Bring Me the Horizon - Shadow Moses"
+    "diablo_ex2_music2.mp3"                     "Bring Me the Horizon - Parasite Eve"
+    "diablo_ex2_music3.mp3"                     "Bring Me the Horizon - Throne"
+    "diablo_s4_ex1.mp3"                         "Bullet For My Valentine - Halo"
```

Comment:
Isn't `Bullet For My Valentine - Halo` a collab with Pendulum as well? I'm referring to [this](https://www.youtube.com/watch?v=B3sn30XQhL4)